specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-07-22-preview/PrometheusRuleGroups.json

```diff
@@ -0,0 +1,601 @@
+          },
+          "default": {
+            "description": "Error response describing the reason for operation failure.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "deprecated": false,
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Resource": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource Id"
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource name"
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource type"
+        },
+        "location": {
+          "type": "string",
+          "description": "Resource location",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        }
+      },
+      "required": [
+        "location"
+      ],
+      "x-ms-azure-resource": true,
+      "description": "An azure resource object"
+    },
+    "PrometheusRuleGroupAction": {
+      "type": "object",
+      "description": "An alert action.",
+      "properties": {
+        "actionGroupId": {
+          "type": "string",
+          "description": "the id of the action group to use."
+        },
+        "actionProperties": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string",
+            "description": "The dictionary of custom properties for the action."
+          },
+          "description": "The properties of an action group object."
+        }
+      }
+    },
+    "PrometheusRuleGroupProperties": {
+      "type": "object",
+      "required": [
+        "scopes",
+        "rules"
+      ],
+      "properties": {
+        "description": {
+          "type": "string",
+          "description": "the description of the Prometheus rule group that will be included in the alert email."
+        },
+        "scopes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "the list of resource id's that this rule group is scoped to."
+        },
+        "interval": {
+          "type": "string",
+          "description": "the interval in which to run the Prometheus rule group."
+        },
+        "rules": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrometheusRule"
+          },
+          "description": "defines the rules in the Prometheus rule group."
+        },
+        "lastUpdatedTime": {
```

Comment:
>lastUpdatedTime [](http://example.com/codeflow?start=9&length=15)

You should be able to remove this if it's the same as the systemData.LastModifiedAt property.

Reply:
Thanks.

Reply:
I don't see an update. Are you planning to remove it?